src/Apps/ArtAdvisor/07-Curated-Discovery/Components/Form/Form.tsx

```diff
@@ -65,6 +67,23 @@ export const Form: React.FC<FormProps> = props => {
 
   const handleSubmit = async () => {
     try {
+      if (_.isEmpty(state.goal)) {
+        sendToast({
+          variant: "error",
+          message: "Please select a goal",
```

Comment:
Maybe debatable, but for demo purposes: goal and interest are required; budget is optional.

Toasts as follows:

![Screenshot 2024-06-28 at 9 05 10 AM](https://github.com/artsy/force/assets/140521/e04ef272-ef9a-4582-a87d-dbd029e98a9b)